library/src/main/java/com/github/twocoffeesoneteam/glidetovectoryou/GlideToVectorYouListener.java

```diff
@@ -1,6 +1,8 @@
 package com.github.twocoffeesoneteam.glidetovectoryou;
 
+import android.graphics.drawable.PictureDrawable;
+
 public interface GlideToVectorYouListener {
     void onLoadFailed();
-    void onResourceReady();
+    void onResourceReady(PictureDrawable pictureDrawable);
```

Comment:
Depending on whether the `PictureDrawable` can be null or not, an annotation here would be helpful. For Kotlin usages

Reply:
Added `@Nullable`